linode/manager
# M3-5324: Clean up remaining CMR files

## Description

Rename all instances of `cmrSpacing` to something more appropriate.

Clean up remaining CMR file versions:

- ColorPalette_CMR 
- dashboard_cmr svg (removed)
- eventMessageGenerator_CMR (to be handled separately)
- LinodeResize_CMR
- SearchBar_CMR styles (removed)
- Tag_CMR

## How to test

Check the `LinodeResize` page and pages containing the `Tag` component to make sure everything is working as expected
